stripe/resource_stripe_price.go

```diff
@@ -564,6 +570,8 @@ func resourceStripePriceCreate(ctx context.Context, d *schema.ResourceData, m in
 				params.Recurring.AggregateUsage = stripe.String(ToString(v))
 			case k == "usage_type" && ToString(v) != "":
 				params.Recurring.UsageType = stripe.String(ToString(v))
+			case k == "meter" && ToString(v) != "":
```

Comment:
Same with the check for empty string.

Reply:
ok, I will remove other occurrences of this pattern in this block as I now realized this is where I picked the pattern from

Reply:
Please remove only your code as there were some specifics in the `price resource` and we don't wanna break it. I'll consolidate it when I'm fixing the reported bug within the price.